projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/methods-and-properties-deprecations.ts

```diff
@@ -1,35 +1,17 @@
 import { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
-import { MethodPropertyDeprecation } from '../../../shared/utils/file-utils';
 import { migrateMethodPropertiesDeprecation } from '../../mechanism/methods-and-properties-deprecations/methods-and-properties-deprecations';
-import { AUTH_HTTP_HEADER_SERVICE_MIGRATION } from './data/auth-http-header.service.migration';
 import { CONFIGURATOR_ATTRIBUTE_HEADER_COMPONENT_MIGRATION } from './data/configurator-attribute-header.component.migration';
 import { CONFIGURATOR_ATTRIBUTE_MULTI_SELECTION_BUNDLE_COMPONENT_MIGRATION } from './data/configurator-attribute-multi-selection-bundle.component.migration';
 import { CONFIGURATOR_ATTRIBUTE_SINGLE_SELECTION_BUNDLE_COMPONENT_MIGRATION } from './data/configurator-attribute-single-selection-bundle.component.migration';
-import { CONFIGURATOR_COMMONS_SERVICE_MIGRATION } from './data/configurator-commons-service.migration';
-import { NAVIGATION_UI_COMPONENT_MIGRATION } from './data/navigation-ui.component.migration';
-import { PROGRESS_BUTTON_COMPONENT_MIGRATION } from './data/progress-button.component.migration';
-import { QUICK_ORDER_SERVICE_MIGRATION } from './data/quick-order.service.migration';
-import { SAVED_CART_EVENT_BUILDER_MIGRATION } from './data/saved-cart-event.builder.migration';
-
-export const METHODS_AND_PROPERTIES_DEPRECATIONS_DATA: MethodPropertyDeprecation[] =
-  [
-    ...AUTH_HTTP_HEADER_SERVICE_MIGRATION,
-    ...QUICK_ORDER_SERVICE_MIGRATION,
-    ...CONFIGURATOR_ATTRIBUTE_MULTI_SELECTION_BUNDLE_COMPONENT_MIGRATION,
-    ...CONFIGURATOR_ATTRIBUTE_SINGLE_SELECTION_BUNDLE_COMPONENT_MIGRATION,
-    ...CONFIGURATOR_COMMONS_SERVICE_MIGRATION,
-    ...CONFIGURATOR_ATTRIBUTE_HEADER_COMPONENT_MIGRATION,
-    ...NAVIGATION_UI_COMPONENT_MIGRATION,
-    ...PROGRESS_BUTTON_COMPONENT_MIGRATION,
-    ...SAVED_CART_EVENT_BUILDER_MIGRATION,
-  ];
+import { METHODS_AND_PROPERTIES_DEPRECATIONS_DATA } from './data/methods-and-properties.migration';
 
 export function migrate(): Rule {
   return (tree: Tree, context: SchematicContext) => {
-    return migrateMethodPropertiesDeprecation(
-      tree,
-      context,
-      METHODS_AND_PROPERTIES_DEPRECATIONS_DATA
-    );
+    return migrateMethodPropertiesDeprecation(tree, context, [
+      ...METHODS_AND_PROPERTIES_DEPRECATIONS_DATA,
+      ...CONFIGURATOR_ATTRIBUTE_MULTI_SELECTION_BUNDLE_COMPONENT_MIGRATION,
```

Comment:
ConfiguratorAttributeMultiSelectionBundleComponent was caught by the breaking change.

Can we add a comment in the breaking-change.json like others to handle this case

Reply:
For now cases like these are present in the breaking change doc, but not in the migration schematics.

I was under the impression that methods and props schematics were meant to address deleted methods and props.  Let's discuss how we addrssess the cases of changed methods and props and if they need to be in the schematics.

Reply:
to discuss, as I'm a bit confused here

Reply:
to discuss~

Reply:
The script now generates migration schematics for updated methods&props (their signature or type changed) in addition to the deleted methods and props.